api/config.go

```diff
@@ -105,6 +107,9 @@ func LoadDefaultConfig(args []string) (config Config, err error) {
 	config.FlagPullInterval = 60
 	config.FlagUpdateHealthReportInterval = 60
 
+	// Set default pull timeout to 10 seconds
+	config.FlagPullTimeout = 10
```

Comment:
yeah 10 sec is probably overkill, we might want to set it to 3 sec as it was before